### PR TITLE
Migrated Polls to Discord Interactions

### LINKED ISSUE #140: Anonymous polls
Add a command to start anonymous polls.

### PR DESCRIPTION
**Pull request by Tert0**
Description

Migrated Polls to Discord Interactions
(Polls are now anonym)
Issue
Closes #140

RIP

### CODE REVIEW

File: general/polls/translations/en.yml [hunk 1]
```diff
@@ -24,3 +25,10 @@ created_by: Created by @{} ({})
 can_not_use_wastebucket_as_option: "You can not use :wastebasket: as option"
 foreign_message: "You are not allowed to add yes/no reactions to foreign messages!"
 could_not_add_reactions: Could not add reactions because I don't have `add_reactions` permission in {}.
+poll_select_placeholder: Select an Option to Vote!
```
Comment (Tristan-H11) [inline general/polls/translations/en.yml]:
```suggestion
poll_select_placeholder: Select an option to vote!
```

File: general/polls/translations/en.yml [hunk 2]
```diff
@@ -24,3 +25,10 @@ created_by: Created by @{} ({})
...
+votes:
+  zero: "{} Votes"
+  one: "{} Vote"
+  many: "{} Votes"
```
Comment (Tristan-H11) [inline general/polls/translations/en.yml]:
```suggestion
  zero: "{} votes"
  one: "{} vote"
  many: "{} votes"
```

**Review by Tristan-H11 — CHANGES_REQUESTED** [review]

File: general/polls/translations/en.yml [hunk 3]
```diff
@@ -24,3 +24,11 @@ created_by: Created by @{} ({})
 can_not_use_wastebucket_as_option: "You can not use :wastebasket: as option"
 foreign_message: "You are not allowed to add yes/no reactions to foreign messages!"
 could_not_add_reactions: Could not add reactions because I don't have `add_reactions` permission in {}.
+poll_select_placeholder: Select an option to vote!
+votes:
+  zero: "{cnt} votes"
```
Comment (Defelo) [inline general/polls/translations/en.yml]: you don't need to specify `zero` if it is the same as `many`

**Review by Defelo — CHANGES_REQUESTED** [review]

File: general/polls/translations/en.yml [hunk 4]
```diff
@@ -27,8 +26,9 @@ foreign_message: "You are not allowed to add yes/no reactions to foreign message
...
 votes:
-  zero: "{} votes"
-  one: "{} vote"
-  many: "{} votes"
+  zero: "{cnt} votes"
+  one: "{cnt} vote"
+  many: "{cnt} votes"
 yes: Yes
 no: No
+option_name_duplicated: "You can't have a duplicated Option!"
```
Comment (Tristan-H11) [inline general/polls/translations/en.yml:33]:
```suggestion
option_name_duplicated: "You cannot have duplicate options!"
```

**Review by Tristan-H11 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**